api/src/opentrons/protocol_engine/state/tips.py

```diff
@@ -48,22 +59,19 @@ def handle_action(self, action: Action) -> None:
                 for column in definition.ordering
                 for well_name in column
             }
-            self._state.column_heads_by_labware_id[labware_id] = set(
-                column[0] for column in definition.ordering
+            self._state.column_by_labware_id[labware_id] = list(
+                column for column in definition.ordering
             )
```

Comment:
Unnecessary call to `list` constructor, prefer `[...]` literals. Also prefer name `columns` to match existing conventions


```suggestion
            self._state.columns_by_labware_id[labware_id] = [
                column for column in definition.ordering
            ]
```